uber/templates/registration/staffers.html

```diff
@@ -8,23 +8,25 @@
 </ol>
 
 {% if search_text %}
-        <div class="panel panel-info">
-    <a href="index?order={{ order }}">Click here</a> to view full volunteer list instead of only search results.
+    <div class="panel panel-info">
+        <a href="index?order={{ order }}">Click here</a> to view full volunteer list instead of only search results.
     </div>
 {% endif %}
 
-<div class="row">
-		<div class="col-md-6">
-<div class="input-group">
-<input class="form-control" type="text" name="filter" id="filter" placeholder="Type Here to Search"/>
-<input type="hidden" name="order" value="{{ order }}" />
-<span class="input-group-addon"><span class="glyphicon glyphicon-search"></span></span>
-</div>
-		</div>
-    <div class="col-md-2">{{ staffer_count }} volunteers</div>
-    <div class="col-md-2">{{ taken_hours }} shift hours taken</div>
-    <div class="col-md-2">{{ total_hours }} shift hours exist</div>
-</div>
+<form method="get" action="staffers">
+    <div class="row">
+        <div class="col-md-6">
+            <div class="input-group">
+                <input class="form-control" type="text" name="search_text" value="{{ search_text }}" placeholder="Type Here to Search"/>
+                <input type="hidden" name="order" value="{{ order }}" />
+                <span class="input-group-addon"><span class="glyphicon glyphicon-search"></span></span>
+            </div>
+        </div>
+        <div class="col-md-2">{{ staffer_count }} volunteers</div>
+        <div class="col-md-2">{{ taken_hours }} shift hours taken</div>
+        <div class="col-md-2">{{ total_hours }} shift hours exist</div>
+    </div>
+</form>
```

Comment:
Excellent question!  In HTML, ``form`` is a block element, which means that anywhere you have a form it will cause a line break.  I originally just wrapped the form around the input elements, but that caused the counts to show up on the next line!

There are other ways around this, e.g.
- we could wrap the form in a div and wrap the counts in a div and have them each given a width or a float or something similar, then wrap them in a parent div
- we could probably set the styling on the form element to have it not be a block element; I've never done this, but I assume it's possible with the right CSS statement

This just seemed like the easiest solution.